packages/components/src/internal/components/search/FilterExpressionView.tsx

```diff
@@ -233,28 +233,32 @@ export const FilterExpressionView: FC<Props> = memo(props => {
                 return (
                     <>
                         <div key="field-value-bool-true">
-                            <input
-                                checked={valueRaw == 'true'}
-                                className=""
-                                type="radio"
-                                name={'field-value-bool' + suffix}
-                                value="true"
-                                disabled={disabled}
-                                onChange={event => updateBooleanFilterFieldValue(filterIndex, event)}
-                            />{' '}
-                            TRUE
+                            <label className="field-value-bool-label clickable">
+                                <input
+                                    checked={valueRaw == 'true'}
+                                    className=""
+                                    type="radio"
+                                    name={'field-value-bool' + suffix}
+                                    value="true"
+                                    disabled={disabled}
+                                    onChange={event => updateBooleanFilterFieldValue(filterIndex, event)}
+                                />{' '}
+                                TRUE
+                            </label>
```

Comment:
I had always thought it best practice to have the label next to instead of surrounding the input element, but this works.